notebooks/en/structured_generation.ipynb

```diff
@@ -0,0 +1,600 @@
+{
```

Comment:
<p>I might just simplify it to the below without adding "naive approach"</p><p> </p><p>"Prompting the model"</p><p> </p><p>"We want to generate these as a JSON <em>dictionary</em> <em>so we can</em> easily parse it for downstream processing (here we will just highlight the source snippets)."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/64/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>